zavod/zavod/meta/assertion.py

```diff
@@ -70,6 +71,20 @@ def parse_metrics(
                 yield from parse_filters(
                     Metric.ENTITY_COUNT, comparison, "country", value
                 )
+            case "property_values" if isinstance(value, dict) and value != {}:
+                for schema_name, props in value.items():
+                    for prop_name, threshold_value in props.items():
+                        yield Assertion(
+                            Metric.ENTITY_COUNT,
```

Comment:
I think this might warrant a different Metric so that the error messages make sense, and also that I'm not sure entity_count makes sense here. But it's been a bit of a job to refresh my memory of how this all fits together. Apologies if this is over-engineered.

https://github.com/opensanctions/opensanctions/blob/main/zavod/zavod/tests/test_validate.py#L79

Reply:
would also be nice to have a test case demonstrating this working, e.g. as per that link